src/Tests/Feature/Web/ThreadStoreTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace TeamTeaTime\Forum\Tests\Feature\Web;
+
+use Illuminate\Foundation\Auth\User;
+use Orchestra\Testbench\Factories\UserFactory;
+use TeamTeaTime\Forum\Database\Factories\CategoryFactory;
+use TeamTeaTime\Forum\Models\Category;
+use TeamTeaTime\Forum\Models\Thread;
+use TeamTeaTime\Forum\Models\Post;
+use TeamTeaTime\Forum\Support\Web\Forum;
+use TeamTeaTime\Forum\Tests\TestCase;
+
+class ThreadStoreTest extends TestCase
+{
+    private string $route = 'thread.store';
+
+    private CategoryFactory $categoryFactory;
+    private UserFactory $userFactory;
+
+    private Category $category;
+    private User $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        
+        $this->categoryFactory = CategoryFactory::new();
+        $this->userFactory = UserFactory::new();
+
+        $this->category = $this->categoryFactory->createOne();
+        $this->user = $this->userFactory->createOne();
+    }
+
+    /** @test */
+    public function should_fail_validation_without_content()
+    {
+        $response = $this->actingAs($this->user)
+            ->post(Forum::route($this->route, $this->category), [
+                'title' => "Thread title",
+                'content' => ""
+            ]);
+
+        $response->assertSessionHasErrors();
+    }
+
+    /** @test */
+    public function should_403_when_not_logged_in()
```

Comment:
Shouldn't it be 401 if not logged in?

Reply:
Currently it 403s because there's no auth middleware applied, so it goes straight through to Authorization - but you're right, I'll change it.

Reply:
Ended up needing to check for a 302 for this since the default `auth` middleware redirects to the `login` route.